certif/tests/integration/components/no-session-panel_test.js

```diff
@@ -23,28 +23,79 @@ module('Integration | Component | no-session-panel', function (hooks) {
 
   module('isMassiveSessionManagementEnabled feature toggle', function () {
     module('isMassiveSessionManagementEnabled feature toggle is true', function () {
-      test('it renders a button link to the session import page', async function (assert) {
-        // given
-        class FeatureTogglesStub extends Service {
-          featureToggles = { isMassiveSessionManagementEnabled: true };
-        }
-        this.owner.register('service:featureToggles', FeatureTogglesStub);
+      module('when certification center is not a type SCO which manages students', function () {
+        test('it renders a button link to the session import page', async function (assert) {
+          // given
+          const store = this.owner.lookup('service:store');
+          const currentAllowedCertificationCenterAccess = store.createRecord('allowed-certification-center-access', {
+            type: 'SUP',
```

Comment:
si on veut représenter un centre SCO dans ce test il faudrait plutôt
``suggestion
            type: 'SCO',
```

Reply:
Le cas présent est "not a type SCO"